spec/prelude/main.md

```diff
@@ -9,6 +9,8 @@ pub use libspecr::prelude::*;
 /// Make the two main modules available.
 pub use crate::{lang, mem};
 
+pub type Offset = Size;
```

Comment:
```suggestion
/// When a non-negative integer is used as an offset into an allocation or type
/// rather than to describe the size of an object or type, use this type instead
/// of `Size` for extra clarity.
pub type Offset = Size;
```